tests/unit/test_s3.py

```diff
@@ -0,0 +1,204 @@
+from unittest.mock import Mock, patch
+
+import pandas as pd
+
+from sdgym.s3 import is_s3_path, parse_s3_path, write_csv, write_file
+
+
+def test_is_s3_path_with_local_dir():
+    """Test the ``sdgym.s3.is_s3_path`` function with a local
+    directory. Expect that the return value is false.
+
+    Input:
+    - path
+
+    Output:
+    - boolean
+    """
+    # setup
+    path = 'path/to/local/dir'
+
+    # run
+    res = is_s3_path(path)
```

Comment:
very minor style comment here: We should try to use the entire variable names whenever possible. So let's call this `result` or `response`, rather than just `res`.